rskj-core/src/main/java/co/rsk/validators/BlockTimeStampValidationRule.java

```diff
@@ -41,27 +41,27 @@ public class BlockTimeStampValidationRule implements BlockParentDependantValidat
 
     private static final Logger logger = LoggerFactory.getLogger("blockvalidator");
 
-    private static final long MAX_TIMESTAMPS_DIFF_IN_SECS = Constants.getMaxTimestampsDiffInSecs();
-
     private final int validPeriodLength;
     private final ActivationConfig activationConfig;
+    private final Constants constants;
     private final TimeProvider timeProvider;
     private final NetworkParameters bitcoinNetworkParameters;
 
-    public BlockTimeStampValidationRule(int validPeriodLength, ActivationConfig activationConfig,
+    public BlockTimeStampValidationRule(int validPeriodLength, ActivationConfig activationConfig, Constants constants,
```

Comment:
Minor thing, but `validPeriodLength` param is being read from `constants` too, although in the caller class.

I'd rather unify the usage either by:
- passing `maxTimestampsDiffInSecs` instead, as it is the only usage for `constants`
- passing only `constants` and reading `validPeriodLength` from it within this class

Reply:
good point.. probably the second option would work, as `constants.getMaxTimestampsDiffInSecs()` depends on a block height